packages/web/src/components/gcds-footer/gcds-footer.css

```diff
@@ -139,44 +170,38 @@
       }
     }
 
-    @media screen and (min-width: 68em) {
-      .brand__container {
-        grid-template-columns: 1fr 1fr 1fr 1fr;
-        grid-template-areas:
-          "list list list wordmark";
-
-        .brand__skip-to-top {
-          display: none;
-        }
-      }
-    }
-
     ul {
       padding: 0;
       margin: 0;
-
-      @media screen and (min-width: 30em) and (max-width: 64em) {
-        column-count: 2;
-      }
+      list-style-type: none;
 
       li {
         display: block;
         margin-right: var(--gcds-spacing-300);
 
+        &:last-child {
+          margin-bottom: 0;
+        }
+
         @media screen and (min-width: 64em) {
           display: inline-block;
         }
 
         &:before {
-          content: "\2022";
-          margin-right: var(--gcds-spacing-300);
+          display: none;
```

Comment:
Since you are only styling the before pseudo-class within the media query now, you should be able to remove this.